docker/build-tools/Dockerfile

```diff
@@ -28,10 +28,10 @@
 # Binary tools
 #############
 
-FROM golang:1.12.5 as binary_tools_context
+FROM golang:1.12.9 as binary_tools_context
 
 # Pinned versions of stuff we pull in
-ENV PROTOBUF_VERSION=3.6.1
```

Comment:
Minor nit might be to alphabetize (or similarly group) the versions. Can be done later if desired.